hydra-node/src/Hydra/Ledger/Cardano.hs

```diff
@@ -352,6 +357,7 @@ genUTxO1 gen = do
 --
 -- NOTE: This generator does
 --  * not produce byron addresses as most of the cardano ecosystem dropped support for that (including plutus),
+--  * not produce too big quantities (determined by 'maximumTotalQuantity')
```

Comment:
Should: this haddock update is redundant now

A _should_ VERY close to _must_ .. so please remove this :)